### DISCUSSION TITLE
Export sorted textboxes

### DISCUSSION THREAD

**Discussion by gandarufu** [xulihang/ImageTrans-docs#740] [Q&A]
I can't find a way to export text boxes, including their sorting and corresponding page.
Ideally, I'd love to end up with a document like this: (json maybe?)
Page 1
Box 1 [x,y,w,h]
Box 2 [x,y,w,h]
...
Page 2
Box 1 [x,y,w,h]
Box 2 [x,y,w,h]
...

Is there a smart way of doing this? The closest was exporting as XLSX, it shows which page the boxes belong to and they are listed in order, but you'd have to run a script to extract the sorting number

**Comment by xulihang** [comment]
Hi,

This is a requirement specifically related to your own needs. If other people can also benefit from this, I can add such an export option.

But currently, you have to make the conversion by yourself.
